### ISSUE TITLE
missing dollar bar source file

### ISSUE THREAD

**Issue by rongzhou** [MizarAI/mizar-labs#19]

**Comment by alexw112358** [comment]
@rongzhou thanks for raising an issue, could you describe in more detail what you were trying to do?

**Comment by rongzhou** [comment]
At first I want to know if you create triple barrier label from tick bar data or dollar bar like I assumed. I find in test directory, there are dollar_bar_dataframe.pkl and tick_dataframe.pk files. I think if you use dollar bar, there should have some source code to create it.

**Comment by alexw112358** [comment]
yes it is possible to create triple barrier labels with the respective transformer (https://github.com/MizarAI/mizar-labs/tree/master/mizarlabs/transformers/targets). We are currently not releasing the source code of the volume/dollar/tick bar, but we are making the volume/dollar/tick bar data publicly available soon for multiple trading pairs.
